projects/packages/videopress/src/client/block-editor/blocks/video/components/banner/style.scss

```diff
@@ -0,0 +1,29 @@
+.block-banner {
+	display: flex;
+    height: 48px;
+    font-size: 14px;
```

Comment:
`line-height` should come together with this one, not?

Reply:
I don't think it's needed since the `display: flex` guarantees vertical alignment. Why should we set the `link-height`?